en/tools/config-catalog-generator-cc/data/router/configs.json

```diff
@@ -0,0 +1,203 @@
+            },
+            {
+              "name": "listenerPort",
+              "type": "string",
+              "required": true,
+              "default": "9090",
+              "description": "Port for listener of Router."
+            },
+            {
+              "name": "securedListenerPort",
+              "type": "string",
+              "required": true,
+              "default": "9095",
+              "description": "Port for secured listener of Router."
+            },
+            {
+              "name": "systemHost",
+              "type": "string",
+              "required": true,
+              "default": "localhost",
+              "description": "The system hostname for system API resources (eg: /testkey and /health)."
+            },
+            {
+              "name": "clusterTimeoutInSeconds",
+              "type": "integer",
+              "required": false,
+              "default": "20",
+              "description": "The timeout for new network connections to hosts in the cluster in seconds."
+            },
+            {
+              "name": "enforcerResponseTimeoutInSeconds",
+              "type": "integer",
+              "required": false,
+              "default": "20",
+              "description": "The timeout for response coming from enforcer to route per API request."
+            }
+          ]
+        }
+      ],
+      "exampleFile": "router.toml"
+    },
+    {
+      "title": "Keystore",
+      "options": [
+        {
+          "name": "keystore",
+          "required": true,
+          "description": "The configurations of key store used in Choreo Connect Adapter",
+          "params": [
+            {
+              "name": "certPath",
+              "type": "string",
+              "required": true,
+              "default": "/home/wso2/security/keystore/mg.pem",
+              "description": "Path of the certificate of the Adaptor"
+            },
+            {
+              "name": "keyPath",
+              "type": "string",
+              "required": true,
+              "default": "/home/wso2/security/keystore/mg.key",
+              "description": "Path of the private key of the Adaptor"
+            }
+          ]
+        }
+      ],
+      "exampleFile": "keystore.toml"
+    },
+    {
+      "title": "CORS",
+      "options": [
+        {
+          "name": "cors",
+          "required": false,
+          "description": "Cors configurations",
+          "params": [
+            {
+              "name": "enabled",
+              "type": "boolean",
+              "required": true,
+              "possible": "true, false",
+              "description": "Enable CORS configurations globally for all endpoints and APIs deployed in Choreo Connect Router"
+            },
+            {
+              "name": "allowOrigins",
+              "type": "list of strings",
+              "required": true,
+              "description": "Allowed origins. set this to [*] allow all origins."
```

Comment:
Suggestion:
`Allowed origins. set this to [*] allow all origins.` -> `Allowed origins. Set this to [*] to allow all origins.`